.github/workflows/jepsen-single-instance.yml

```diff
@@ -0,0 +1,52 @@
+name: jepsen-single-instance
+
+on: [push, pull_request]
```

Comment:
Even single instance test in not needed for each push. IMVHO

Reply:
Arguments?